Admin/RedirectAdmin.php

```diff
@@ -73,6 +73,12 @@ public function configureViews(ViewCollection $viewCollection): void
         $listToolbarActions = [
             new ToolbarAction('sulu_admin.add'),
             new ToolbarAction('sulu_admin.delete'),
+            new ToolbarAction('sulu_admin.upload', [
+                'routeName' => 'sulu_redirect.import',
+                'label' => 'sulu_redirect.import',
+                'maxFiles' => 1,
```

Comment:
not sure if I would add any limit or instead go with nothing or `multiple => true`

Reply:
Since `react-dropzone` supports the `maxFiles` out of the box, I would prefer also using `maxFiles` here and calculate the value of `multiple` ... that increases the flexibility of the toolbar action at no cost

Reply:
`multiple` is also an option and we sure should support that. I think that is more important as to support `maxFiles`:

https://github.com/react-dropzone/react-dropzone/blob/bb42b940281f9b17642911c7988bc69d822d1f7e/src/index.js#L57